tests/lib.rs

```diff
@@ -46,8 +46,8 @@ fn data_map_content_only(){
   assert!(dm.has_chunks() == false);
 }
 
-fn random_string(length: u64) -> String {
-  (0..length).map(|_| (0x20u8 + (rand::random::<f32>() * 96.0) as u8) as char).collect()
+fn random_bytes(length: u64) -> Vec<u8> {
+  (0..length).map(|_| (0x20u8 + (rand::random::<f32>() * 96.0) as u8) as u8).collect()
```

Comment:
in random_bytes it for sure already makes sense to use `usize`, but we will have to come back to this after v0.1.0 to use memory mapped files to allow for bigger file sizes... I think :)